backend/api/src/db/session.rs

```diff
@@ -0,0 +1,61 @@
+use chrono::{DateTime, Utc};
+use sqlx::PgConnection;
+use uuid::Uuid;
+
+use crate::token::SessionMask;
+
+#[must_use]
+fn generate_session_token() -> String {
```

Comment:
I would generate 32 _bytes_ of data and then base64 encode it. You're limiting the entropy here with just `Alphanumeric`.